src/harness/client.ts

```diff
@@ -739,7 +739,7 @@ namespace ts.server {
             return notImplemented();
         }
 
-        getEncodedSemanticClassifications(_fileName: string, _span: TextSpan): Classifications {
+        getEncodedSemanticClassifications(_fileName: string, _span: TextSpan, _format?: "original" | "2020"): Classifications {

```

Comment:
I would use a const enum by the time you’re ready to make a real PR

Reply:
const string enum* — strings are still a good idea since it’s originating outside of TypeScript